Lista2/Ex1/ex1-Part2/main.c

```diff
@@ -0,0 +1,163 @@
+#include <stdio.h>
+#include <stdlib.h>
+
+typedef struct stack{
+    int data;
+    struct stack *link;
+} Stack;
+
```

Comment:
You forget to put the header struct, in the question is mentioned that need the header struct.

Reply:
Vou colocar

Reply:
perfeito